src/components/Signup/Signup.test.js

```diff
@@ -277,7 +277,97 @@ describe('Signup', () => {
+
+    const search = '?utm_content=test-utm-content';
+    const location = { search, pathname: `/signup/${search}` };
+
+    // Act
+    const { container } = render(
+      <AppServicesProvider forcedServices={dependencies}>
+        <DopplerIntlProvider locale="en">
+          <Router>
+            <Signup location={location} />
+          </Router>
+        </DopplerIntlProvider>
+      </AppServicesProvider>,
+    );
+
+    act(() => {
+      const inputName = container.querySelector('input#firstname');
+      fireEvent.change(inputName, { target: { value: values.firstname } });
+
+      const inputLastname = container.querySelector('input#lastname');
+      fireEvent.change(inputLastname, { target: { value: values.lastname } });
+
+      const inputPhone = container.querySelector('input#phone');
+      fireEvent.change(inputPhone, { target: { value: values.phone } });
+
+      const inputEmail = container.querySelector('input#email');
+      fireEvent.change(inputEmail, { target: { value: values.email } });
+
+      const inputPassword = container.querySelector('input#password');
+      fireEvent.change(inputPassword, { target: { value: values.password } });
+
+      const inputPrivacyPolicies = container.querySelector('input#accept_privacy_policies');
+      fireEvent.click(inputPrivacyPolicies);
+    });
+
+    act(() => {
+      const submitButton = container.querySelector('button[type="submit"]');
+      fireEvent.submit(submitButton);
+    });
+
+    // Assert
+    await waitFor(() => expect(registerUser).toHaveBeenCalledTimes(1));
+    expect(registerUser).toHaveBeenCalledWith({
+      ...values,
+      email: values['email'].trim(),
+      accept_promotions: '',
+      firstOrigin: undefined,
+      captchaResponseToken: 'hardcodedResponseToken',
+      language: 'en',
+      origin: 'login',
+      redirect: '',
+      utm_source: 'direct',
+      utm_campaign: null,
+      utm_cookies: [],
+      utm_medium: null,
+      utm_term: null,
+      gclid: null,
+      utm_content: 'test-utm-content',
+    });
+  });
+
+  it('should show that any extra param is added or harcoded', async () => {
+    // Arrange
+    const values = {
+      firstname: 'Juan',
+      lastname: 'Perez',
+      phone: '+54 223 655-8877',
+      email: ' hardcoded@hardcoded.com',
+      password: 'HarcodedPass123',
+      accept_privacy_policies: true,
+    };
+    const registerUser = jest.fn().mockImplementation((values, statusForm) => ({
+      success: false,
+      expectedError: {
+        emailAlreadyExists: true,
+      },
+    }));
+    const dependencies = {
+      ...defaultDependencies,
+      dopplerLegacyClient: {
+        ...defaultDependencies.dopplerLegacyClient,
+        registerUser,
+      },
+      utmCookiesManager: {
+        setCookieEntry: jest.fn(),
+        getUtmCookie: jest.fn().mockImplementation(() => []),
+      },
+    };
+
+    const search = '';
+    const location = { search, pathname: `/signup/${search}` };
```

Comment:
```suggestion
    const location = { pathname: `/signup/` };
```

Reply:
@aferraromedina I think this will work too, can you try?

Reply:
yes!

Reply:
Before we had 2 tests, one for utm_content empty and another one with utm_content null, now I see only one O_o

Reply:
yes, empty utm_content test was equal to the existing test "should show that the email with whitespace already exists when submit" so i deleted it.